### DISCUSSION TITLE
FR: Ability to hide the Title field column on the CP Entry Listing

### DISCUSSION THREAD

**Discussion by adrianjean** [craftcms/cms#13261] [Ideas]
- I have an entry type for `Person` with `First Name` and `Last Name` fields. 
- I auto generate the `Title` field in the entry form combining First and Last names.
- On the CP Entry listing for that section, it would be nice to hide the `Title` field column since it's redundant as I have First and Last name columns showing for easy sorting.

I feel like this might have been asked/discussed — so I searched past requests and found this issue, but I think this is more related to data entry than the entry listing on the Control Panel:  https://github.com/craftcms/cms/issues/6110

**Comment by brandonkelly** [comment]
What would link to the entry edit page, then?

**Reply by adrianjean** [reply]
Good question.
I supposed, whatever field is selected as the first field — but.... what if someone selects the "Link" field as the fist... how is that handled.

What if, the first column after the selection checkbox is an ever present Pencil-square icon that always has the link. This way the data columns are separate from the functional columns?

Column "Entry" would then be named "Title" as one of the selectable columns.

**Reply by adrianjean** [reply]
<img width="467" alt="Screenshot 2023-05-30 at 3 46 51 PM" src="https://github.com/craftcms/cms/assets/3161182/ecd4f722-f226-415f-9af2-a42eb9b7bec9">

Quick Mockup — "Home" would not be a link tho.
